common/ripple/userUtils.py

```diff
@@ -332,247 +248,202 @@ async def ban(userID: int) -> None:
 
+    # Set user as restricted in db
+    await glob.db.execute(
+        "UPDATE users SET privileges = privileges & %s, "
+        "ban_datetime = UNIX_TIMESTAMP() WHERE id = %s",
+        [~privileges.USER_PUBLIC, user_id],
+    )
 
-async def unrestrict(userID: int) -> None:
-    """
-    Unrestrict userID.
-    Same as unban().
+    # Notify bancho about this ban
+    await glob.redis.publish("peppy:ban", user_id)
 
-    :param userID: user id
-    :return:
-    """
 
-    await unban(userID)
+async def unrestrict(user_id: int) -> None:
+    """Unrestrict a user by id. Functionally equivalent to calling unban()."""
+    await unban(user_id)
 
 
-async def appendNotes(
-    userID: int,
+async def append_cm_notes(
+    user_id: int,
     notes: str,
-    addNl: bool = True,
-    trackDate: bool = True,
+    add_newline: bool = True,
+    track_date: bool = True,
 ) -> None:
     """
-    Append `notes` to `userID`'s "notes for CM"
+    Append to a given user's "notes for community management".
 
-    :param userID: user id
+    :param user_id: user id
     :param notes: text to append
-    :param addNl: if True, prepend \n to notes. Default: True.
-    :param trackDate: if True, prepend date and hour to the note. Default: True.
+    :param add_newline: if True, prepend \n to notes. Default: True.
+    :param track_date: if True, prepend date and hour to the note. Default: True.
     :return:
     """
 
-    if trackDate:
+    if track_date:
         notes = f"[{strftime('%Y-%m-%d %H:%M:%S', localtime())}] {notes}"
 
-    if addNl:
+    if add_newline:
         notes = f"\n{notes}"
 
     await glob.db.execute(
         "UPDATE users " 'SET notes = CONCAT(COALESCE(notes, ""), %s) ' "WHERE id = %s",
-        [notes, userID],
+        [notes, user_id],
     )
 
 
-async def getPrivileges(userID: int) -> int:
-    """
-    Return `userID`'s privileges
-
-    :param userID: user id
-    :return: privileges number
-    """
-
+async def get_privileges(user_id: int) -> int:
+    """Return a user's privileges."""
     result = await glob.db.fetch(
         "SELECT privileges " "FROM users " "WHERE id = %s",
-        [userID],
+        [user_id],
     )
 
     return result["privileges"] if result else 0
 
 
-async def getFreezeTime(userID: int) -> int:
-    """
-    Return a `userID`'s enqueued restriction date.
-
-    :param userID: userID of the target (restrictee)
-    :return: timestamp
-    """
-
+async def get_freeze_restriction_date(user_id: int) -> int:
+    """Return a user's enqueued restriction date."""
     result = await glob.db.fetch(
         "SELECT frozen FROM users WHERE id = %s",
-        [userID],
+        [user_id],
     )
-
     return result["frozen"] if result else 0
 
 
-async def getFreezeReason(userID: int) -> Optional[str]:
-    result = await glob.db.fetch(
-        "SELECT freeze_reason FROM users WHERE id = %s",
-        [userID],
-    )
-    return result["freeze_reason"] if result["freeze_reason"] else None

```

Comment:
Seems that the freeze message sent to users has been made static since the addition of this function & db column.

I've removed all code referencing `freeze_reason`, and removed it from the payments-service, the only other in-use system to reference this column.

Reply:
We can have a follow-up task to drop the column from the users table.